packages/aws-cdk-lib/aws-certificatemanager/lib/private-certificate.ts

```diff
@@ -75,11 +85,17 @@ export class PrivateCertificate extends CertificateBase implements ICertificate
     // Enhanced CDK Analytics Telemetry
     addConstructMetadata(this, props);
 
+    let certificateExport: string | undefined;
+    if (props.allowExport !== undefined) {
+      certificateExport = props.allowExport ? 'ENABLED' : 'DISABLED';
+    }
```

Comment:
The [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/TemplateReference/aws-resource-certificatemanager-certificate.html#cfn-certificatemanager-certificate-certificateexport) says:

> If you do not specify an export preference in a new CloudFormation template, it is the same as explicitly denying export of your certificate.

Based on this, there is no value in differentiating between DISABLED and undefined (i.e. not providing a value)

Reply:
See the same comment I made here: https://github.com/aws/aws-cdk/pull/35079#discussion_r2300610838, for the similar condition in the `certificate.ts` file.

Reply:
```suggestion
const certificateExport = (props. allowExport === true) ? 'ENABLED' : undefined
```